pygmt/__init__.py

```diff
@@ -40,7 +40,7 @@ def print_clib_info():
     Includes the GMT version, default values for parameters, the path to the
     ``libgmt`` shared library, and GMT directories.
     """
-    from .clib import Session
+    from pygmt.clib import Session
```

Comment:
> This PR is almost done, but we have this error:
> 
> ```
> pygmt/__init__.py:43:4: C0415: Import outside toplevel (pygmt.clib.Session) (import-outside-toplevel)
> ```
> 
> @weiji14 Do you have any idea what it means?

Hmm, not sure why it complains here. I think this complaint means `from pygmt.clib import Session` should be placed at the top (outside of this `print_clib_info` function/module, but I thought we allowed `clib` to be imported at any level though (see https://github.com/GenericMappingTools/pygmt/pull/484/commits/4e76530efafacc95fd9d3dad497977ddbc9493ea#r446575346). Might need to disable the pylint error for this line

```suggestion
    from pygmt.clib import Session  # pylint: disable=import-outside-toplevel
```

Reply:
Changing the `.pylintrc` to:
```
allow-any-import-level=pygmt.clib,
                        pygmt.clib.Session,
```
works.

Should we change the `.pylintrc` instead?

Reply:
What version of pylint are we on now? I can't quite make out what my thinking was at https://github.com/GenericMappingTools/pygmt/pull/484#discussion_r446575346...

Reply:
`import-outside-toplevel` was introduced in [pylint 2.4.0](http://pylint.pycqa.org/en/latest/whatsnew/changelog.html#what-s-new-in-pylint-2-4-0), but it may have [a bug](https://github.com/PyCQA/pylint/issues/3175), which was [fixed in pylint 2.4.3](http://pylint.pycqa.org/en/latest/whatsnew/changelog.html#what-s-new-in-pylint-2-4-3).

Reply:
> What version of pylint are we on now?

The CI is using pylint 2.6.0.

Reply:
The following changes work well for pylint 2.6.0:
```diff
-allow-any-import-level=clib,
-                       clib.Session,
+allow-any-import-level=pygmt.clib.Session,
```

Reply:
Ah good, go with that then.